packages/mg-imagescraper/package.json

```diff
@@ -26,6 +26,8 @@
   "dependencies": {
     "cheerio": "^1.0.0-rc.2",
     "got": "11.6.1",
-    "lodash": "4.17.20"
+    "lodash": "4.17.20",
+    "mime": "^2.4.6",
```

Comment:
When adding dependencies, we install them with `yarn` and also use the `--exact` option to [not have the caret](https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004). You will also need to include the `yarn.lock` file when working with dependencies. We skip the `--exact` and work with the `^` only for modules where we're 100% certain that we can/should always update them, e. g. all `@tryghost` packages 😃